### PR TITLE
Updated plunker service to specify versions of NG + RXJS.

### PR DESCRIPTION
**Pull request by Blackbaud-BobbyEarl**
This makes test https://github.com/blackbaud/skyux2/pull/1491 easier to test.  I should mention, I wasn't able to solve the `takeUntil` error on the `list-view-grid`.  Line 194 is an attempt to quickly solve that in the docs (for now).

To test:

- `npm install -g serve` (or similar package).  The latest version of `serve` inadvertently made node >= 7.6 a requirement.  For now, I would run `npm install -g serve@6.5.3`.  https://github.com/zeit/serve/issues/348
- In skyux repo
  - `npm run build`
  - `serve . --cors --ssl --cache 0` (or similar command)
  - Open `https://localhost:5000` and approve SSL cert
- In skyux2-docs repo
  - `skyux serve`

I did evaluate StackBlitz, and think that will be a logical move for us, but am waiting on https://github.com/stackblitz/core/issues/437 before investing too much more time.  I've saved my work at https://github.com/blackbaud/skyux2-docs/tree/stackblitz-service until then.

### CODE REVIEW

File: src/app/components/shared/demo-page-plunker-service.ts [hunk 1]
```diff
@@ -178,6 +189,10 @@ ${imports.join('\n')}
 
 import { AppComponent } from './app.component';
 
+// This is temporary to get list-view-grid functioning properly
+// A future relese of SKY UX components would fix this.
```
Comment (Blackbaud-SteveBrush) [inline src/app/components/shared/demo-page-plunker-service.ts]: "relese"

Also, might be inconsequential, but would this comment appear in the plunker's source files? Again, not super important, just curious if we want this language on our plunkers.

File: src/app/components/shared/demo-page-plunker-service.ts [hunk 2]
```diff
@@ -45,38 +56,40 @@ export class SkyDemoPagePlunkerService {
       ...files,
       {
         name: 'config.js',
-        content:
-`System.config({
+        content: `
+var ngVersion = '@${VERSION.full}';
```
Comment (Blackbaud-SteveBrush) [inline src/app/components/shared/demo-page-plunker-service.ts]: Neat!

**Comment by Blackbaud-SteveBrush** [comment]
Might be worth noting: the NPM package `serve` requires Node.JS version v7.6 and above (uses `async`).

**Comment by Blackbaud-BobbyEarl** [comment]
Good catch on the `serve` version requirement.  I updated the original PR comment to mention that and addressed the typo.

**Pull request merged**